setup.sh

```diff
@@ -17,7 +17,7 @@ set -eu
 
 _error_report() {
   echo >&2 "Exited [$?] at line $(caller):"
-  pr -tn $0 | tail -n+$(($1 - 3)) | head -n7 | sed "4s/^\s*/>>> /"
+  cat -n $0 | tail -n+$(($1 - 3)) | head -n7 | sed "4s/^\s*/>>> /"
```

Comment:
I know the original isn't commented, but can you add a bit of what this is supposed to do. It feels like not a great pattern to make folks process what the goal is of this line.

Reply:
Done (above the method)